src/popup/utils/constants.js

```diff
@@ -107,6 +107,7 @@ export const MAX_AMOUNT_WITHOUT_CONFIRM = 10;
 export const AENS_DOMAIN = '.chain';
 export const MAX_AUCTION_NAME_LENGTH = 12 + AENS_DOMAIN.length;
 export const MIN_NAME_LENGTH = 14;
+export const AUTO_EXTEND_NAME_BLOCKS_INTERVAL = 100;
```

Comment:
Let's maybe use some interval between blocks when we will be asking for extending, and I also call it only on extension opening